src/app/add-item/add-item.component.ts

```diff
@@ -15,10 +14,12 @@ export class AddItemComponent implements OnInit {
   })
   inputItem: string = '';
   productLength: number = 0;
-  products:any = [];
+  products = Object({});
```

Comment:
Its should be an interface

Reply:
Provided Object type as the interface is not getting assigned properly from the response Object.